libiop/bcs/pow.tcc

```diff
@@ -93,8 +93,9 @@ hash_digest_type pow<FieldT, hash_digest_type>::solve_pow_internal(
 
     size_t num_words = pow.length() / sizeof(size_t);
     size_t pow_int = 0;
-    while (this->verify_pow(node_hasher, challenge, pow) == false)
+    while (!this->verify_pow(node_hasher, challenge, pow))
     {
+        // printf("Trying %zx\n", pow[(num_words - 1)*sizeof(size_t)]);
```

Comment:
Debug info. Should this line be removed?

Reply:
Whoops